src/site/shared/containers/ContextMenu/index.tsx

```diff
@@ -55,7 +58,47 @@ export const ContextMenu = ({info, paste}: Props) => {
             else if (ev.type === "mousedown")
                 dispatch(CloseContextMenu());
         });
-    }, [input])
+    }, [input]);
+
+    /*Position changes are calculated using the react hook so that the
+     *context menu does not jump around during other update events.
+    */
```

Comment:
nit: use `//` for non-JSDoc comments
nit: link the issue for reference like ("`necessary to fix #914`")